scripts/debug_run.py

```diff
@@ -110,6 +110,15 @@ def grab_graphs(args):
       fd.write(report)
 
 
+def metrics_to_tensorboard(args):
+  script_path = os.path.join(get_scripts_path(), 'metrics_to_tensorboard.py')
+  log_dir = os.path.join(args.outdir, 'tensorboard')
+  report = subprocess.check_output([
```

Comment:
We can just use `subprocess.run` since we don't use output here right?

Reply:
I agree. I can change it after I get all the feedback.

Reply:
Updated to subprocess.run.